lib/createTopLevelExpect.js

```diff
@@ -1604,6 +1604,10 @@ expectPrototype.child = function() {
     parent.addAssertion(testDescription, handler, childExpect);
     return this;
   };
+  childExpect.exportTheme = function(...args) {
+    parent.installTheme(...args);
```

Comment:
This will work, but I don't think it's a totally correct implementation if we use `exportAssertion`, `exportStyle`, and `exportType` as analogies.

The parent expect should be able to install its own theme (before or after the `child` is created), which should take precedence when the parent expect is used, and also be available as a fallback when the child expect is used.

Getting it to work like that will probably require some work in magicpen itself.

Reply:
Hmm, sorry, I think I'm confusing myself. I think this is actually exactly what exporting a theme should do, sorry.